tests/TestCase/Http/Cookie/CookieTest.php

```diff
@@ -475,6 +475,14 @@ public function testCreateFromHeaderString(): void
         $this->assertNull($result->getSameSite());
     }
 
+    public function testCreateFromHeaderStringEmptyValue(): void
+    {
+        $header = 'cakephp; expires=Wed, 01-Dec-2027 12:00:00 GMT; path=/; domain=cakephp.org;';
+        $result = Cookie::createFromHeaderString($header);
+
+        $this->assertSame('', $result->getValue());
+    }
```

Comment:
Can we label this as malformed so future devs know it wasn't some misinterpretation of the standard and remove the support?

Reply:
I don't know if we'll be able to remove support as there will always be bad servers out in the wild.